.travis.yml

```diff
@@ -10,9 +10,5 @@ addons:
 script:
   - sonar-scanner
   
-branches:
-  only:
-  - /.*/
-  
```

Comment:
Oh yeah, no mention about branches means also that all branches are built. Good catch.